docs/modules/ROOT/pages/index.adoc

```diff
@@ -3,3 +3,15 @@
 openshift4-splunk-forwarder is a Commodore component to manage openshift4-splunk-forwarder.
 
 See the xref:references/parameters.adoc[parameters] reference for further details.
+
+== Setup
```

Comment:
You should move this section into a `docs/modules/ROOT/pages/how-tos/getting-started.adoc` and actually explain what needs to be done with the self-signed certificate and key.

The missing steps to actually get the component up and running are (at least, from what I figured out during the review):
* Enable the component for the cluster (instantiated if necessary)
* Generate secret and store in Vault for `splunk.sharedkey`
* Store Splunk HEC token in Vault
* Store SSL private key (and passphrase if any) in Vault
* Configure SSL certificate in `splunk.ssl.cert`